examples/example_resources/lib/gen/assets.gen.dart

```diff
@@ -84,7 +88,7 @@ class AssetGenImage {
     bool matchTextDirection = false,
     bool gaplessPlayback = false,
     bool isAntiAlias = false,
-    String? package = 'example_resources',
+    @deprecated String? package = package,
```

Comment:
No deprecate message?

Reply:
This is because of a `dart_style` formatting issue and I cannot get a consist behavior when generating files with `@Deprecated`, so go for the `@deprecated`.

Reply:
> but is the `dart_style` package still needed? `dart format` works fine without it.

So far it looks to be required because codes are generated when running runners and produced formatted files without calling `dart format`.

Reply:
Oh I see, dart_style is used during the generating of the code.

I noticed the version of dart_style listed in packages/core/pubspec.yaml is a year old, and the dart format being run in the GitHub actions seems a lot newer than that.

Reply:
> I noticed the version of dart_style listed in packages/core/pubspec.yaml is a year old, and the dart format being run in the GitHub actions seems a lot newer than that.

That's fine unless it released v3 then we'll need to manually bump it.